OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh - Unit test case for SetUpFinalDayPopUp

# Description

Unit test case for SetUpFinalDayPopUp


## How to test:

1. check into current branch
2. do npm install
3. npm test SetUpFinalDayPopUp.test.js
4. verify that all tests pass

## Screenshots or videos of changes:

<img width="1012" alt="Screenshot 2023-12-22 at 4 17 37 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/f43e9904-87df-499a-825d-e97de8a84cd4">
